carmin-server/server/test/resources/test_executions.py

```diff
@@ -133,3 +137,53 @@ def test_get_with_executions(self, test_config):
         executions, error = ExecutionSchema(many=True).load(json_response)
         assert not error
         assert len(executions) == number_of_executions
+
+    def test_get_with_offset(self, test_config, number_of_executions):
+        offset = 6
+        response = test_config.test_client.get(
+            '/executions?offset={}'.format(offset),
+            headers={
+                "apiKey": standard_user().api_key
+            })
+        json_response = load_json_data(response)
+        assert len(json_response) == number_of_executions - offset
+
+    def test_get_with_invalid_offset(self, test_config, number_of_executions):
+        offset = "invalid"
+        response = test_config.test_client.get(
+            '/executions?offset={}'.format(offset),
+            headers={
+                "apiKey": standard_user().api_key
+            })
+        error = error_from_response(response)
+
+        expected_error_code_and_message = copy.deepcopy(
+            INVALID_QUERY_PARAMETER)
+        expected_error_code_and_message.error_message = expected_error_code_and_message.error_message.format(
+            offset, 'offset')
+        assert error == expected_error_code_and_message
+
+    def test_get_with_limit(self, test_config, number_of_executions):
+        limit = 4
+        response = test_config.test_client.get(
+            '/executions?limit={}'.format(limit),
+            headers={
+                "apiKey": standard_user().api_key
+            })
+        json_response = load_json_data(response)
+        assert len(json_response) == limit
+
+    def test_get_with_invalid_limit(self, test_config, number_of_executions):
```

Comment:
Please add a test for invalid limits int values like -1 or a value greater than the count.